test/thrift/generator/service_test.exs

```diff
@@ -370,8 +370,6 @@ defmodule Thrift.Generator.ServiceTest do
 
   thrift_test "clients exit if they try to use a closed client", ctx do
     Process.flag(:trap_exit, true)
-
-    ref = Process.monitor(ctx.server)
```

Comment:
@fishcakez , I've only taken a cursory look, but isn't the server a `thrift_socket_server` instance here? Ranch shouldn't be involved.